man/systemd.special.xml

```diff
@@ -503,8 +514,11 @@
       <varlistentry>
         <term><filename>sysinit.target</filename></term>
         <listitem>
-          <para>A special target unit covering early boot-up
-          scripts.</para>
+          <para>This target pulls in the services required for system
+          initialization. System services pulled in by this target should
+          be prepared to work without all mount points, and should declare
```

Comment:
"non-root" file system paths is a difficult thing to say.

Maybe we should say this all differently: after sysinit.target the system is configured far enough so that socket units, timer units, path units may be configured correctly, i.e. all local file systems and a select few important remote fs are mounted. What precisely an "important" remote fs is might be something we might not even mention here... /home is not of this kind, but /var/tmp, /var and /tmp really are... We are fine with booting up without /home around, but certainly not without the other three... But of course, this is only half a fix so far, because some people might want to split out only /var/log or so, which we do not explicitly cover for right now...